tower-grpc-examples/src/helloworld/server.rs

```diff
@@ -42,8 +41,8 @@ pub fn main() {
 
     let new_service = server::GreeterServer::new(Greet);
 
-    let h2_settings = Default::default();
-    let mut h2 = Server::new(new_service, h2_settings, DefaultExecutor::current());
+    let mut server = Server::new(new_service);
+    let http = Http::new().http2_only(true).clone();
```

Comment:
Is this clone required?

Reply:
I couldn't figure out how to get a non &mut ref since http2_only returns a &mut self. Clone worked do go from a &self to a Self. Any pointers around this would be helpful.

Reply:
Oh yea...

```rust
let mut http = Http::new();
http.http2_only(true);
```

Reply:
bleh, can we just add some sort of `build()` to those builder types?